guides/common/modules/proc_adding-azure-connection.adoc

```diff
@@ -12,6 +12,11 @@ To use the CLI instead of the web UI, see the xref:cli-adding-azure-connection_{
 . In the *Name* field, enter a name for the compute resource.
 . From the *Provider* list, select *Azure Resource Manager*.
 . Optional: In the *Description* field, enter a description for the resource.
+. By default, the *Cloud* is set to Public/Standard. 
+  If you are using the Azure Government Cloud then the following regions are supported in the dropdown:
+  - US Goverment
+  - China
+  - Germany
```

Comment:
```suggestion
** Germany
```

Same for China and `US Government` (currently missing an `n`)

Reply:
Why not just `*` ?